CrewChiefV4/GameState/GameStateData.cs

```diff
@@ -1730,10 +1742,29 @@ public void SetNextDeltaPoint(float distanceRoundTrack, float speed, DateTime no
                 deltaPoints[nextDeltaPoint] = now;
                 currentDeltaPoint = nextDeltaPoint;
             }
+
         }
-        public DateTime GetCurrentDeltaPointTime()
+
+        public float GetDeltaTime(DeltaTime playerDelta)
         {
-            return deltaPoints[currentDeltaPoint];
+            TimeSpan splitTime = new TimeSpan(0);
+            if (playerDelta.deltaPoints.Count > 0 && deltaPoints.Count > 0)
+            {
+                //opponent is behind
+                if (distanceRoundTrack < playerDelta.distanceRoundTrack)
+                {
+                    splitTime = playerDelta.deltaPoints[currentDeltaPoint] - deltaPoints[currentDeltaPoint];
+                }
+                else if (distanceRoundTrack > playerDelta.distanceRoundTrack)
+                {
+                    splitTime = playerDelta.deltaPoints[playerDelta.currentDeltaPoint] - deltaPoints[playerDelta.currentDeltaPoint];
+                }
+                else
+                {
+                    return 0f;
+                }
+            }
+            return Math.Abs((float)splitTime.TotalSeconds);
         }
```

Comment:
shouldn't this be rounded to hundredths?

Reply:
This does aswell

Reply:
aye, from the API docs: "This number might include whole and fractional seconds"